scripts/changelog-generator.js

```diff
@@ -10,13 +10,33 @@
  * @property {{ login: string }=} author
  */
 
+/**
```

Comment:
Feel free to just write this thing in TypeScript proper if you want

Reply:
Probably makes sense by now 😅